### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L interm,mot,nd,ro,sie,sur,tthe,warmup`

### CODE REVIEW

File: deeplabcut/pose_tracking_pytorch/model/backbones/vit_pytorch.py [hunk 1]
```diff
@@ -346,12 +346,12 @@ def resize_pos_embed(posemb, posemb_new, hight, width):
...
         "Resized position embedding from size:{} to size: {} with height:{} width: {}".format(
-            posemb.shape, posemb_new.shape, hight, width
+            posemb.shape, posemb_new.shape, height, width
         )
     )
     posemb_grid = posemb_grid.reshape(1, gs_old, gs_old, -1).permute(0, 3, 1, 2)
-    posemb_grid = F.interpolate(posemb_grid, size=(hight, width), mode="bilinear")
-    posemb_grid = posemb_grid.permute(0, 2, 3, 1).reshape(1, hight * width, -1)
+    posemb_grid = F.interpolate(posemb_grid, size=(height, width), mode="bilinear")
+    posemb_grid = posemb_grid.permute(0, 2, 3, 1).reshape(1, height * width, -1)
```
Comment (luzpaz) [inline deeplabcut/pose_tracking_pytorch/model/backbones/vit_pytorch.py:354]: Changes in `deeplabcut/pose_tracking_pytorch/model/backbones/vit_pytorch.py` are source changes

File: deeplabcut/pose_estimation_tensorflow/training.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def train_network(
 
     max_snapshots_to_keep: int or None
         Sets how many snapshots are kept, i.e. states of the trained network. Every
-        saving interation many times a snapshot is stored, however only the last
+        saving iteration many times a snapshot is stored, however only the last
         ``max_snapshots_to_keep`` many are kept! If you change this to None, then all
         are kept.
```
Comment (luzpaz) [inline deeplabcut/pose_estimation_tensorflow/training.py:83]: This whole paragraph is grammatically confusing

**Review by MMathisLab — APPROVED** [review]
